features/liquidity/components/LiquidityBreakdown.tsx

```diff
@@ -73,13 +74,37 @@ const PoolHeader = ({ tokenA, tokenB, priceBreakdown }: PoolHeaderProps) => (
   </Inline>
 )
 
+const SwapFee = ({
+  protocolFee,
+  lpFee,
+}: {
+  protocolFee?: number
+  lpFee?: number
+}) => {
+  const protocolFeeSafe = protocolFee != undefined ? protocolFee : 0
+  const lpFeeSafe = lpFee != undefined ? lpFee : 0.3
```

Comment:
isnt this the same as setting `lpFee = 0.3` and not making it optional? we'll always be passing `lpFee` when we use it right even if `lpFee ` is undefined i think

Reply:
yes I think `lpFee = 0.3` is cleaner in the default. But still waiting on the tooltip for `undefined` cases https://github.com/Wasmswap/wasmswap-interface/pull/260#discussion_r972535790. I think we should just keep the same tooltip tbh and switch to default parameters.

Reply:
What do you think @ben2x4?

Reply:
what is the exact question here? I'm cool with whatever y'all think is the cleaner style

Reply:
Ok sounds good. Thanks!